openshift/sandboxed-containers-operator
# Patch series to update Golang version and fix bundle creation for dev

**- Description of the problem which is fixed/What is the use case**
Build and deploy of operator bundle from sources fails

**- What I did**
Fixed the image name, added dev instructions and bumped up Golang

**- How to verify it**
Instructions to build and deploy operator by following the [guide](docs/DOCUMENTATION.md) should work

**- Description for the changelog**

- Update Golang version and fix bundle creation for dev
